### PR TITLE
After game page HTML+CSS

### PR DESCRIPTION
**Pull request by Ale282313**

### CODE REVIEW

File: app/static/css/after_game.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.statistics {
+    font-size: 18px;
+    color: rgb(36, 60, 36);
+
+}
```
Comment (simutoni) [inline app/static/css/after_game.css]: please add a blank line between classes.

File: app/templates/game/after_game.html [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+        <div class="col-md-8 col-md-offset-2 stats-wrapper">
+        <div class="row">
+            <div class="col-md-4 col-md-offset-2"></div>
+            <div class="col-md-4 col-md-offset-1 statistics">Lorem ipsum dolor</div>
+        </div>
+        <div class="row">
+            <div class="col-md-4 col-md-offset-3 player">PlayerX :</div>
+            <div class="col-md-4 statistics">Lorem ipsum dolor</div>
+        </div>
+        <div class="row">
```
Comment (simutoni) [inline app/templates/game/after_game.html]: what is the meaning of this rows? I see that is a lot of duplicate code.

Comment (Ale282313) [inline app/templates/game/after_game.html]: I used the rows to make a table with 2 asymmetrical columns and 7 rows, to show statistics after the game.

File: app/static/css/after_game.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.statistics {
+    font-size: 18px;
+    color: rgb(36, 60, 36);
+
```
Comment (cristimihok) [inline app/static/css/after_game.css]: Unnecessary empty line

File: app/static/css/after_game.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.player {
+    font-size: 22px;
+    color: rgb(36, 60, 36);
```
Comment (cristimihok) [inline app/static/css/after_game.css]: Use rgb or hash colors. Don't mix them. I saw that hash are mostly used in the other files.

File: app/static/css/after_game.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.empty {
+    min-height: 20px;
+}
+
+.stats-wrapper {
+    /*margin-top: 40px;*/
```
Comment (cristimihok) [inline app/static/css/after_game.css]: Remove if not used.

File: app/static/css/after_game.css (hunk 4 above)
Comment (Ale282313) [inline app/static/css/after_game.css]: We will use rgb :) in the hole project

**Pull request merged**